app/scripts/lib/setupSentry.js

```diff
@@ -238,7 +238,7 @@ function getSentryEnvironment() {
 
 function getSentryTarget() {
   if (
-    getManifestFlags().sentry?.doNotForceSentryForThisTest ||
+    !getManifestFlags().sentry?.forceEnable ||
```

Comment:
We need to change the `||` here to `&&` because we flipped the meaning of the flag